### PR TITLE
Update BouncyCastle NuGet package.

### PR DESCRIPTION
**Pull request by Gymbotan**

### CODE REVIEW

File: TuviSRPLib/TuviSRPLib/ExtendedHashDigest.cs [hunk 1]
```diff
@@ -40,17 +41,17 @@ public void BlockUpdate(byte[] input, int inOff, int length)
                 throw new ArgumentOutOfRangeException(nameof(inOff), "Parameter inOff can not be negative.");
             }
 
-            if (length < 0)
+            if (inLen < 0)
             {
-                throw new ArgumentOutOfRangeException(nameof(length), "Parameter length can not be negative.");
+                throw new ArgumentOutOfRangeException(nameof(inLen), "Parameter length can not be negative.");
```
Comment (al-kau) [inline TuviSRPLib/TuviSRPLib/ExtendedHashDigest.cs:46]: `Parameter length can not be negative.`
length should be replaced with inLen

File: TuviSRPLib/TuviSRPLib/ExtendedHashDigest.cs [hunk 2]
```diff
@@ -27,8 +28,8 @@ public ExtendedHashDigest()
         /// </summary>
         /// <param name="input">The byte array containing the data.</param>
         /// <param name="inOff">The offset into the byte array where the data starts.</param>
-        /// <param name="length">the length of the data.</param>
-        public void BlockUpdate(byte[] input, int inOff, int length)
+        /// <param name="inLen">the length of the data.</param>
+        public void BlockUpdate(byte[] input, int inOff, int inLen)
```
Comment (al-kau) [inline TuviSRPLib/TuviSRPLib/ExtendedHashDigest.cs:32]: Why do you remane it?

File: TuviSRPLib/TuviSRPLibTests/TuviSRPLibTests.csproj [hunk 3]
```diff
@@ -21,9 +21,15 @@
...
-    <PackageReference Include="Portable.BouncyCastle" Version="1.9.0" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="BouncyCastle.Cryptography" Version="2.2.1" ExcludeAssets="Compile" GeneratePathProperty="true" />
+    <Reference Include="BouncyCastle.Cryptography">
+      <HintPath>$(PkgBouncyCastle_Cryptography)\lib\netstandard2.0\BouncyCastle.Cryptography.dll</HintPath>
+    </Reference>
+  </ItemGroup>
+
```
Comment (ValeraFinebits) [inline TuviSRPLib/TuviSRPLibTests/TuviSRPLibTests.csproj:32]: Could you please add comment about this workaround and issue link?

File: TuviSRPLib/TuviSRPLib/ExtendedHashDigest.cs (hunk 1 above)
Comment (Gymbotan) [reply TuviSRPLib/TuviSRPLib/ExtendedHashDigest.cs]: done

File: TuviSRPLib/TuviSRPLibTests/TuviSRPLibTests.csproj (hunk 3 above)
Comment (Gymbotan) [reply TuviSRPLib/TuviSRPLibTests/TuviSRPLibTests.csproj:34]: done

**Review by ValeraFinebits — APPROVED** [review]

**Review by anatolse — APPROVED** [review]

**Pull request merged**